checkstyle-suppressions.xml

```diff
@@ -5,4 +5,9 @@
     <suppress files="generated-test-sources" checks=".*"/>
     <suppress files="com.cerner.beadledom.client.resteasy.ApacheHttpClient4Dot3Engine" checks=".*"/>
     <suppress files="com.cerner.beadledom.jaxrs.PATCH" checks="AbbreviationAsWordInName" />
+
+    <suppress files="com.cerner.beadledom.jooq.DSLContextProvider"
```

Comment:
Why not `DslContextProvider`?

Reply:
Since this class name starts with the name of an existing class from jooq -- `DSLContext` -- we wanted to keep the casing the same even though it's different from how we would traditionally name our classes.